profiles/api/ping.py

```diff
@@ -1,13 +1,16 @@
 from flask import Blueprint, make_response
 from werkzeug.wrappers import Response
 
-PING_BP = Blueprint('ping', __name__)
 
+def create_blueprint() -> Blueprint:
+    blueprint = Blueprint('ping', __name__)
 
-@PING_BP.route('/ping')
-def ping() -> Response:
-    response = make_response('pong')
-    response.headers['Content-Type'] = 'text/plain; charset=UTF-8'
-    response.headers['Cache-Control'] = 'must-revalidate, no-cache, no-store, private'
+    @blueprint.route('/ping')
+    def _ping() -> Response:
```

Comment:
Run into https://github.com/pallets/flask/issues/2434. Don't want to disable the check, but means all routes with have to begin with `_`.

Reply:
:+1: as not ideal but not wrong: immediately clear that this identifier is not available at the module level but only inside this scope

Reply:
Does get used inside `url_for()` though.